### PR TITLE
bayesCor option names

### PR DESCRIPTION
**Pull request by AlexanderLyNL**
Changed option names for Simon

### CODE REVIEW

**Comment by AlexanderLyNL** [comment]
This will probably work once https://github.com/jasp-stats/jaspRegression/pull/151 is merged

**Review by Kucharssim — APPROVED** [review]

File: inst/Description.qml [hunk 1]
```diff
@@ -8,7 +8,7 @@ Description
 	description		: qsTr("Apply common Bayesian tests from sufficient statistics")
 	icon			: "analysis-bayesian-ttest.svg"
 	requiresData	: false
-	version			: "0.15"
```
Comment (Kucharssim) [inline inst/Description.qml:11]: can you also change the version in `DESCRIPTION` accordingly?

**Review by Kucharssim — CHANGES_REQUESTED** [review]

File: inst/Description.qml (hunk 1 above)
Comment (Kucharssim) [reply inst/Description.qml:11]: I was too quick approving the PR at first :D

Comment (AlexanderLyNL) [reply inst/Description.qml:11]: Yes definitely + done.

**Review by Kucharssim — APPROVED** [review]

**Pull request merged**